.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@1.0.0
+  artifactory-orb: jfrog/artifactory-orb@1.0.1
+
+executors:
+  builder:
+    docker:
+      - image: cimg/python:3.10.6      
+    working_directory: ~/project
+
+jobs:
+  Build and Publish:
+    executor: builder
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - python/install-packages:
+          pkg-manager: poetry
+      - run:
+          name: Run pytest
+          command: |
+            poetry run pytest
+      - run:
+          name: Build python wheel
+          command: |
+            poetry build -f wheel
+      - run:
+          name: Build Docker image
+          command: |
+            docker build -t coverahealth.jfrog.io/development-docker/chip-gateway-hl7-listener:1.0.0 .
```

Comment:
Will we update this at some point so it's not always `1.0.0`?

Reply:
Yes this is temporary. Once we have a standard CircleCI config template and base build image, this will resolve itself.

Reply:
Wunderbar!